test/Test/Prolens/Property.hs

```diff
@@ -66,3 +67,33 @@ eqFun :: Fun Maybe Int Int -> Fun Maybe Int Int -> PropertyT IO ()
 eqFun fun1 fun2 = do
     x <- forAll genInt
     unFun fun1 x === unFun fun2 x
+
+monoidalSpec :: Spec
+monoidalSpec = describe "Monoidal" $ do
+    describe "(->)" $ do
+        it "Identity: pappend pempty id ≡ id" $ hedgehog $ do
+            f <- forAllWith (const "f") genFunction
+            x <- forAll genInt
+            y <- forAll genInt
+            pappend pempty f (x, y) === (x, f y)
```

Comment:
`id` can't be used, and you actually check another law here, which is correct 👍🏼 
However, we can make it even a bit more elegant and general, if we use the fact that it is `Strong`. We can check these properties instead:

```haskell
pappend f pempty ≡ first f
pappend pempty f ≡ second f
```